### ISSUE TITLE
Adding a SORT event

### ISSUE THREAD

**Issue by innopal** [bencripps/react-redux-grid#49]
I'd like to be able to keep the last state that grid was sorted on in my redux store.

Something like:
```

{
  dataIndex: 'name',
  direction: 'DESC'
}
```

**Comment by bencripps** [comment]
Working on this one now -- should be up soon.

**Comment by bencripps** [comment]
K, give it a test and let me know!

**Comment by innopal** [comment]
Perfect! I'll let you know the result asap.

**Comment by innopal** [comment]
@bencripps 
It doesn't hit that event!
Doesn't get to debugger when I click on any column to sort!

Here's my code: 
```
export const events = {
  HANDLE_ROW_CLICK: () => {},
  HANDLE_BEFORE_SORT: ({ property, direction, store, column }) => {
    debugger;
  }
};
```

**Comment by innopal** [comment]
I installed master and I can see your code for this new event added there when I dig into node_modules.

**Comment by bencripps** [comment]
This should fix it; forgot to run the `dist` build.

**Comment by innopal** [comment]
Nice work!
Any release plan in the works?

**Comment by bencripps** [comment]
published as 2.3.2
